### PR TITLE
Creating an animated counter in JavaScript

### LINKED ISSUE #1107: How to create an animated counter in JavaScript
### Brief Summary: 
We will learn to create an animated counter for a webpage using JavaScript

### Key Takeaways: 
At the end, the reader will learn about:
- The Animation Frame in JavaScript
- Timestamps and their use in counters
- How the `Math.min` and `Math.floor` methods are also used in counters

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by Agusioma**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1107

### CODE REVIEW

**Comment by kayere** [comment]
@Agusioma - Please avoid making changes to the PR (unless asked to). This might disrupt the review process.

**Comment by Agusioma** [comment]
Noted. @kayere

File: articles/animation-counter-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+We got three user-defined functions:
+
+- `animate()`
+- `step()`
+- `load()`
+
+The `animate()` function is a higher-order function containing the `step()` function.
+
+```Javascript
+function animate(obj, initVal, lastVal, duration) {
```
Comment (kayere) [inline articles/animation-counter-javascript/index.md:74]: Remove this redundant code snippet.

**Review by kayere — CHANGES_REQUESTED** [review]
Hello @Agusioma, I have just gone through your article. Everything works perfectly fine. However, I'd suggest you restructure the article. I can see you only have one subheading; which is not appealing. To add to that, the article's word count (excluding code snippets) is less than 750. Consider adding more content (like conclusion)to satisfy the minimum word count requirement. Revise the article and notify me when you have made the changes.

**Comment by Agusioma** [comment]
Noted. Thank you

File: articles/animation-counter-javascript/index.md (hunk 1 above)
Comment (Agusioma) [reply articles/animation-counter-javascript/index.md]: Done

**Comment by Agusioma** [comment]
@kayere I have done the requested changes. Please check

**Review by kayere — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by louisefindlay23 — APPROVED** [review]
@hectorkambow, this is ready for publication.

**Comment by kayere** [comment]
Hello @louisefindlay23, I think you have made changes to the wrong PR👀🤔

**Comment by hectorkambow** [comment]
@Agusioma 
Noticing that there is multiple index files on this PR - not sure if it was something we did on our side or if the PR was created like this

**Comment by hectorkambow** [comment]
> Hello @louisefindlay23, I think you have made changes to the wrong PR👀🤔

@kayere  

Louise has been helping go through PR for final reviews. That may be why she was making some edits. 

One thing I noticed though is this PR has multiple index file (more than one article).

**Comment by Agusioma** [comment]
@hectorkambow it's on your side because right now am not seeing my article here

**Comment by louisefindlay23** [comment]
That's it corrected, @hectorkambow and @kayere.

**Comment by kayere** [comment]
> > Hello @louisefindlay23, I think you have made changes to the wrong PReyesthinking
> 
> @kayere
> 
> Louise has been helping go through PR for final reviews. That may be why she was making some edits.
> 
> One thing I noticed though is this PR has multiple index file (more than one article).

Oow ...I see.😎

The article was fine at the time of my review. I think the commits that followed resulted to that.

**Comment by kayere** [comment]
> That's it corrected, @hectorkambow and @kayere.

Awesome

**Comment by Agusioma** [comment]
@louisefindlay23 it's like you deleted the wrong article. It was the one belonging to this PR.

**Pull request merged**